docs/dlk/limitation.md

```diff
@@ -0,0 +1,122 @@
+# Limitation
+
+## DLK Data Types
+### Supported data type
```

Comment:
Theses names comes from TensorFlow internal type name.
We assume document readers are not familiar with TF internal.